### ISSUE TITLE
examples/text/news_topic_classification.py - has it been tested? 

### ISSUE THREAD

**Issue by jkang1640** [learnables/learn2learn#379]
Hello, 

I tried the script but the loss / accuracy does not get improve till the end of training and it seems that the model didn't learn anything.. 

Looking closely into the code, it seems that validation set is wrongly used as a query set...  (train dataset is not divided into a support and query set. Instead, validation set is used as a query set and loss on the validation set is used to update the model) 

Has this code been tested already before release? If so, did you observe that the the model accuracy improves as the training goes?

Thank you.

**Comment by seba-1511** [comment]
Hello @jkang1640,

Indeed, we were never able to get MAML to work well on text. See #96 for details.

This makes me think I should remove the text example, since it seems to be below the standard of the library. I'll leave the issue open until it's removed. If you want to submit a PR to fix the script (or remove it) I'll happily merge it.

**Comment by seba-1511** [comment]
Update: I've decided to remove it in #400 since there isn't much interest. Closing this issue now.
